CHANGELOG.md

```diff
@@ -3,6 +3,7 @@
 ## master (unreleased)
 
 ### New Features
+- ci: add support for arm64 binary releases
```

Comment:
@williballenthin @heywoodlh should we consider this a breaking change since the artifact names are changing? This will cause issues for anyone downloading releases using direct URLs...

Reply:
Would you prefer we keep the same naming layout for the AMD64 executables and only add `arm64` to the ARM64 releases?

It's too bad there's no way (that I know of) to have some sort of filename alias in GitHub releases.

Reply:
Yes, I think leaving the amd64 artifact names unchanged is probably best for now. Otherwise, we shouldn't merge this until we're much closer to a major release.

Reply:
> for now I'll just revert to the original names.

This is done. :smile: